### PR TITLE
Adding intro section to Tutorials archive

### LINKED ISSUE #1131: Add intro section to Tutorials archive
The Lesson Plans and Courses pages now have matching intro sections, so we should add the same thing to the Tutorials archive. This will create some consistency across content types on the site.

### PR DESCRIPTION
**Pull request by hlashbrooke**
Adds intro section with custom text to bring Tutorials archive in line with other content types - fixes #1131

### CODE REVIEW

File: wp-content/themes/pub/wporg-learn-2020/archive-wporg_workshop.php [hunk 1]
```diff
@@ -25,7 +25,6 @@
 	<section>
 		<div class="row align-middle between section-heading section-heading--with-space gutters">
 			<?php the_archive_title( '<h1 class="section-heading_title h2 col-8">', '</h1>' ); ?>
```
Comment (adamwoodnz) [inline wp-content/themes/pub/wporg-learn-2020/archive-wporg_workshop.php]: I don't think we need to handle a row layout here anymore

```suggestion
		<div class="section-heading section-heading--with-space">
			<?php the_archive_title( '<h1 class="section-heading_title h2">', '</h1>' ); ?>
```

Comment (adamwoodnz) [reply wp-content/themes/pub/wporg-learn-2020/archive-wporg_workshop.php]: `col-12` can be removed from line 31 also

Comment (hlashbrooke) [reply wp-content/themes/pub/wporg-learn-2020/archive-wporg_workshop.php]: OK - I copied this from the courses archive page, so I assume it was all needed, but happy to remove it if it's not necessary here.

**Review by adamwoodnz — APPROVED** [review]
Nicely done! :shipit:

**Pull request merged**